src/LinearOperators.jl

```diff
@@ -24,20 +24,23 @@ const LinearOperatorIndexType = Union{UnitRange{Int}, StepRange{Int, Int}, Abstr
 # import methods we overload
 import Base.eltype, Base.isreal, Base.size, Base.show
 import Base.+, Base.-, Base.*
-@static if VERSION < v"0.6.0-"
-  import Base.(.+), Base.(.-), Base.(.*)
-end
 import Base.A_mul_B!, Base.At_mul_B!, Base.Ac_mul_B!
-import Base.transpose, Base.ctranspose
+import Base.transpose
+@static if VERSION < v"0.7.0-"
+  import Base.ctranspose
+else
+  import Base.adjoint
+  import LinearAlgebra.diag
```

Comment:
This causes an error on OSx nightly. Is it supposed to be satisfied by `using Compat.LinearAlgebra`? Otherwise, it should appear in `REQUIRE`, but only for version 0.7.

Reply:
I restarted Travis and it worked, they must've updated something in the mean time.
I'm gonna fix the conflict, rebase and hope that appveyor also pass.